README.md

```diff
@@ -34,15 +49,18 @@ default of 8.4) you have to set this attribute as a node override:
 Attributes
 -----------
 
-`['pg-multi']['master']` : sets the IP address that defines the master node
+`['pg-multi']['master']` : sets the IP address that defines the master node (Required)
 
 `['pg-multi']['slaves']` : is any array that defines the IP address(es) of
-the slave node(s).
+the slave node(s). (Required)
 
 `['pg-multi']['replication']['user']` : allows for the setting of a custom name for
 the replication user, by default it is set to 'repl'.
 
-`['pg-multi']['replication']['password'] ` : allows for the setting of a custom password for the replication user.
+`['pg-multi']['replication']['password'] ` : allows for the setting of a custom password for the replication user. (Required)
```

Comment:
The libraries don't use attributes, though, do they? They appear to have their own parameters with different names (e.g. `PgReplUser` and `PgSlave` use a parameter called `repl_pass`). I don't see `node['pg-multi']['replication']['password']` used anywhere in the libraries.